lib/apollo-federation/schema.rb

```diff
@@ -12,30 +12,27 @@ def self.included(klass)
     end
 
     module CommonMethods
-      FEDERATION_2_PREFIX = <<~SCHEMA
-        extend schema
-          @link(url: "https://specs.apollo.dev/federation/v2.0")
-
-      SCHEMA
+      DEFAULT_LINK_NAMESPACE = 'federation'.freeze
+      SUPPORTED_FED2_VERSIONS = ['2.0', '2.1', '2.2', '2.3'].freeze
```

Comment:
for a code first* library IMHO it does not make sense to support multiple fed 2 versions -> if you use the library with the v2, you probably should just default to the latest

*in SDL first where we load the provided file it might be needed